packages/cache/types/index.d.ts

```diff
@@ -1,8 +1,32 @@
 // Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
 // TypeScript Version: 2.2
-import { EmotionCache } from '@emotion/utils'
+import { SerializedStyles } from '@emotion/serialize'
+import { StyleSheet } from '@emotion/sheet'
 
-export { EmotionCache }
+export interface RegisteredCache {
+  [key: string]: string
+}
+
+export interface EmotionStyleSheet extends StyleSheet {
+  constructor: typeof StyleSheet
+}
+
+export interface EmotionCache {
+  inserted: {
+    [key: string]: string | true
+  }
+  registered: RegisteredCache
+  sheet: EmotionStyleSheet
```

Comment:
This is the only thing I can think of that would solve the `constructor` issue, let me know if you don't want it or if you have any thoughts